### ISSUE TITLE
Support endpoint authentication

### ISSUE THREAD

**Issue by RinkeHoekstra** [CLARIAH/grlc#91]
As it is quite common to have restricted access to SPARQL endpoints, there should be an easy way to set HTTP Basic authentication parameters in the config file that are then passed through SPARQL HTTP requests for the default SPARQL endpoint.

**Comment by c-martinez** [comment]
Good point @RinkeHoekstra -- we could consider going back to using SPARQLWrapper and use their http auth (`wrapper.setCredentials(user, passwd)`). As I remember (from #33) the issue with SPARQLWrapper was lack of support for content type plaintext. I'll look into it.

**Comment by RinkeHoekstra** [comment]
You can also pass auth=(user,pass) to a requests get or post.
